.github/workflows/main.yml

```
@@ -28,3 +28,28 @@ jobs:
           export PATH="/usr/share/miniconda/bin:$PATH"
           source activate black
           pyflakes spython/oci spython/image spython/instance spython/main
+
+  pytest:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: eWaterCycle/setup-singularity@v6
```

This is ready to go pending one request - I'm a bit conservative about using actions that I don't control (and that aren't under GitHub's umbrella). Could you please update this to be the commit associated with the release and then comment for the tag? e.g.,:
```suggestion
      - uses: eWaterCycle/setup-singularity@1631b12cf3878381179be0eab9624219bc12979e # v6 release
```
Looked up: https://github.com/eWaterCycle/setup-singularity/commit/1631b12cf3878381179be0eab9624219bc12979e

---

done